src/dungeon_data.c

```diff
@@ -59,12 +59,15 @@ struct Dungeon *get_players_dungeon_f(const struct PlayerInfo *player,const char
 
 struct Dungeon *get_dungeon_f(PlayerNumber plyr_num,const char *func_name)
 {
-    if ((plyr_num < 0) || (plyr_num >= DUNGEONS_COUNT))
+    if ((plyr_num >= 0) || (plyr_num < DUNGEONS_COUNT))
+    {
+        return &(game.dungeon[(int)plyr_num]);
+    }
+    if (plyr_num != NEUTRAL_PLAYER) // Suppress error for never existing but valid neutral 'player'
```

Comment:
@OrsonMaxwell, could you please write a regular message in the log when this occurs? So instead of the error say a lvl3 log message, and leave the error for players >5.